content/sensu-go/5.12/installation/auth.md

```diff
@@ -1094,6 +1099,412 @@ example      | {{< highlight shell >}}"name": "activedirectory"{{< /highlight >}
+    },
+  	"metadata": {
+    	"name": "oidc"
+  	}
+	}
+}
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+### RBAC for OIDC definition specification
+
+#### `oidc` attributes
+
+additionalScopes | 
+-----------------|------
+description      | Scopes to include in the claims, in addition to the default `email`, `openid` and `profile` scopes. _NOTE: only required for certain OIDC providers, such as Okta._
+required         | false
+type             | Array
+example          | {{< highlight shell >}}"additionalScopes": [ "groups" ]{{< /highlight >}}
+
+clientId     | 
+-------------|------
+description  | The OIDC provider application "Client ID" _NOTE: requires [registration of an application in the OIDC provider][4]._
+required     | true
+type         | String
+example      | {{< highlight shell >}}"clientId": "a8e43af034e7f2608780"{{< /highlight >}}
+
+clientSecret | 
+-------------|------
+description  | The OIDC provider application "Client Secret" _NOTE: requires [registration of an application in the OIDC provider][4]._
+required     | true
+type         | String
+example      | {{< highlight shell >}}"clientSecret": "b63968394be6ed2edb61c93847ee792f31bf6216"{{< /highlight >}}
+
+insecure     | 
+-------------|------
+description  | Determines whether or not to skip SSL certificate verification (e.g. for self-signed certificates).
+required     | false
+type         | Boolean
+default      | false
+example      | {{< highlight shell >}}"insecure": true{{< /highlight >}}
+
+redirectURL  | 
+-------------|------
+description  | Redirect URL to provide to the OIDC provider. _NOTE: only required for certain OIDC providers, such as Okta._
+required     | false
+type         | String
+example      | {{< highlight shell >}}"redirectURL": "http://localhost:4000/login/callback"{{< /highlight >}}
+
+server       | 
+-------------|------
+description  | The location of the OIDC server you wish to authenticate against.
+required     | true
+type         | String
+example      | {{< highlight shell >}}"server": "https://localhost:9031"{{< /highlight >}}
+
+roles        | 
+-------------|------
+description  | An array of`roles` definitions.
+required     | true
+type         | Array
+example      | {{< highlight shell >}}"roles": [
+  {
+    "name": "guests",
+    "members": [
+      "myorganization/guests"
+    ],
+    "datacenters": [
+      "us-west-1"
+    ],
+    "subscriptions": [
+      "webserver"
+    ],
+    "readonly": true
+  },
+  {
+    "name": "operators",
+    "members": [
+      "myorganization/operators"
+    ],
+    "datacenters": [],
+    "subscriptions": [],
+    "readonly": false
+  }
+]
+{{< /highlight >}}
+
+#### `roles` attributes
+
+Please see the [RBAC definition specification][25] for information on how to
+configure RBAC roles.
+
+## Register an OIDC Application
+
+To use OIDC for authentication requires registration of your Sensu Enterprise
+Dashboard as an "application". Please note the following instructions to
```

Comment:
It's not only for the dashboard now; in fact for 5.12 we only support CLI login. The interface in the WebUI is coming later.